src/index.tsx

```diff
@@ -40,6 +43,17 @@ if (isSentryEnabled()) {
   })
 }
 
+// Placeholder API key. Actual API key used in the proxy server
+const ANALYTICS_DUMMY_KEY = '00000000000000000000000000000000'
+const ANALYTICS_PROXY_URL = process.env.REACT_APP_AMPLITUDE_PROXY_URL
+const COMMIT_HASH = process.env.REACT_APP_GIT_COMMIT_HASH
+initializeAnalytics(ANALYTICS_DUMMY_KEY, OriginApplication.INTERFACE, {
+  proxyUrl: ANALYTICS_PROXY_URL,
+  defaultEventName: SharedEventName.PAGE_VIEWED,
+  commitHash: COMMIT_HASH,
+  isProductionEnv: isProductionEnv(),
+})
+
```

Comment:
I moved this initialization from `App.tsx` so that the values are availbe to the `StatsigProvider` in this file. We could put `StatsigProvider` at the lower level too, but for some reason that caused issues with e2e tests

Reply:
I think this moreso depends on what types of flags are being used. If it's anything related to startup / loading, I could see the flag provider being at the outermost level being important. If just for features within, I don't think it's a huge concern

Reply:
I don't think you're using any of these values in StatsigProvider rn, do we anticipate using them there in future PRs?

Reply:
@lynnshaoyu I am, via the `getDeviceId` function of the analytics package. this requires amplitude to be initialized